src/routes/authRouter.ts

```diff
@@ -1,6 +1,10 @@
 import express, { Router } from 'express';
 const passport = require('passport');
```

Comment:
```suggestion
import passport from 'passport'
```

import 형식으로 변경해야 할 것 같아요!

Reply:
적용했습니다! 감사합니다!